### ISSUE TITLE
Map RouteData on Request Type with get request

### ISSUE THREAD

**Issue by mdissel** [Kahbazi/MediatR.AspNetCore.Endpoints#6]
Any tips how to map values from the Url and/or querystring on the Request Type object?

```
public class ViewEntityRequest : IRequest<ViewEntityRequest.Response> {

		public long Id { get; set; }

		public string Type { get; set; }
}

[HttpGet("view/{type}/{id}")]
```

Sample get request with url: /view/entity/5

Thanks
Marco

**Comment by Kahbazi** [comment]
There's no model binding right now beside deserializing from body. 
Your pull request is a good start to add this feature. 
For the future I'm waiting on this issue https://github.com/dotnet/aspnetcore/issues/27347, which move model binding out of MVC, so it can be used in middleware level.

**Comment by Kahbazi** [comment]
This is now available in [2.0.0.preview.2](https://www.nuget.org/packages/MediatR.AspNetCore.Endpoints/2.0.0-preview.2).
